tutorial_sarscov2.md

```diff
@@ -0,0 +1,259 @@
+---
+layout: default
+permalink: /tutorial/sars-cov2/
+---
+
+This tutorial shows the basics of how to interact with V-pipe.
+
+For the purpose of this Tutorial, we will work with the `sars-cov2` branch which is [adapted for the SARS-CoV-2 virus]({{ "/sars-cov-2/" | relative_url }}).
+
+> **Organizing Data**:
+>
+> V-pipe expects [the input samples to be organized in a two-level hierarchy](https://github.com/cbg-ethz/V-pipe/wiki/getting-started#input-files):
+>
+> - At the first level, input files grouped by samples (e.g.: patients or biological replicates of an experiment).
+> - A second level for distinction of datasets belonging to the same sample (e.g.: sample dates).
+> - Inside that directory, the sub-directory `raw_data` holds the sequencing output in FASTQ format (optionally compressed with GZip).
+> - When in split files, paired-ends reads need to have `_R1` and `_R2` suffixes in their name.
```

Comment:
If paired-end reads are used, we expect separate files for the time being

Reply:
(By the way, in the next iteration, depending on what solution we pick to address the "_`sort` over runs the TMP storage and crashes_" bug, we might solve the interleaved files situation "bird" with the same stone.)

Reply:
Not necessarily, because we merge replicates in this step